src/app/app.module.ts

```diff
@@ -17,6 +17,11 @@ import { FlashcardComponent } from './pages/flashcard/flashcard.component';
 import { MessagesComponent } from './pages/messages/messages.component';
 import { AddCardComponent } from './pages/add-card/add-card.component';
 import { ForumsComponent } from './pages/forums/forums.component';
+import { CameraViewComponent } from './pages/camera-view/camera-view.component';
+import { ContinuePageComponent } from './pages/continue-page/continue-page.component';
+import { FinalConfirmComponent } from './pages/final-confirm/final-confirm.component';
+import { SettingsComponent } from './pages/settings/settings.component';
+import { EditCardComponent } from './pages/edit-card/edit-card.component';
```

Comment:
I like the names for these bottom 2 components.  The first three (CameraView, ContinuePage, FinalConfirm) I'm not exactly sure what those components are for without looking into the code files. More descriptive component names could be helpful.

For example if cameraView is for taking the picture then maybe ImageCapture component... if it's for looking at pictures then PhotoView Component. etc etc.

Reply:
renamed CameraView/ContinuePage/FinalConfirm to ImageCapture/NoPicture/ConfirmCompletedCard